### ISSUE TITLE
LetsEncrypt v1 deprecation notice

### ISSUE THREAD

**Issue by webdesserts** [beakerbrowser/homebase#9]
When I run the latest version of homebase I get the following warning:

```txt
==========================================================
==                greenlock.js (v2.2.0+)                ==
==========================================================

Please specify 'version' option:

        'draft-11' for Let's Encrypt v2 and ACME draft 11
        ('v02' is an alias of 'draft-11'

or

        'v01' for Let's Encrypt v1 (deprecated)
         (also 'npm install --save le-acme-core' as this legacy dependency will soon be removed)

This will be required in versions v2.3+


Detected Let's Encrypt v01 URL (deprecated). Changing version to v01.

=== WARNING ===

Let's Encrypt v1 is deprecated.
Please update to Let's Encrypt v2 (ACME draft 11)

Success! Bound to port '80' to handle ACME challenges and redirect to https
Success! Serving https on port '443'
(node:21362) [DEP0096] DeprecationWarning: timers.unenroll() is deprecated. Please use clearTimeout instead.
```

Also seems to be a deprecation notice for the usage of `timers.unenroll()`. Not sure if that's something you have control over.

**Comment by pfrazee** [comment]
Okay, we should probably just set the version then.

**Comment by pfrazee** [comment]
Published `1.1.5`
